store/storetest/webhook_store.go

```diff
@@ -412,38 +346,28 @@ func testWebhookStoreGetOutgoingList(t *testing.T, ss store.Store) {
 
 	o2, _ = ss.Webhook().SaveOutgoing(o2)
 
-	if r1, err := ss.Webhook().GetOutgoingList(0, 1000); err != nil {
-		t.Fatal(err)
-	} else {
-		hooks := r1
-		found1 := false
-		found2 := false
-
-		for _, hook := range hooks {
-			if hook.CreateAt != o1.CreateAt {
-				found1 = true
-			}
-
-			if hook.CreateAt != o2.CreateAt {
-				found2 = true
-			}
-		}
+	r1, err := ss.Webhook().GetOutgoingList(0, 1000)
+	require.Nil(t, err)
+	hooks := r1
+	found1 := false
+	found2 := false
 
-		if !found1 {
-			t.Fatal("missing hook1")
+	for _, hook := range hooks {
+		if hook.CreateAt != o1.CreateAt {
+			found1 = true
 		}
-		if !found2 {
-			t.Fatal("missing hook2")
-		}
-	}
 
-	if result, err := ss.Webhook().GetOutgoingList(0, 2); err != nil {
-		t.Fatal(err)
-	} else {
-		if len(result) != 2 {
-			t.Fatal("wrong number of hooks returned")
+		if hook.CreateAt != o2.CreateAt {
+			found2 = true
 		}
 	}
+
+	require.True(t, found1, "missing hook1")
+	require.True(t, found2, "missing hook2")
+
+	result, err := ss.Webhook().GetOutgoingList(0, 2)
+	require.Nil(t, err)
+	require.Equal(t, 2, len(result), "wrong number of hooks returned")
```

Comment:
You can use here `require.Len` instead of `require.Equal`, which is more semantic.